docker/docker-compose-postgres.yml

```diff
@@ -25,6 +25,13 @@ services:
       - ../:/candlepin-dev
     depends_on:
       - db
+    secrets:
+      - sonar_crt
+
+secrets:
+  sonar_crt:
+    file: /run/container_secrets/sonar.crt
```

Comment:
this file doesn't exist on the server, i can add this to the jenkins nodes if you'd like